src/3d/qgs3dmapexportsettings.cpp

```diff
@@ -41,3 +43,10 @@ Qgs3DMapExportSettings::~Qgs3DMapExportSettings()
   settings.setValue( QStringLiteral( "UI/last3DSceneExportExportNormals" ), mExportNormals );
   settings.setValue( QStringLiteral( "UI/last3DSceneExportExportTextures" ), mExportTextures );
 }
+
+QString Qgs3DMapExportSettings::exportFilePath() const
```

Comment:
can you rename to exportFileUrl instead, as that's what it's actually returning?